### DISCUSSION TITLE
Datumsspalte in Tabelle

### DISCUSSION THREAD

**Discussion by AndyCooler** [public-ui/kolibri#1940] [Q&A] [answered]
Hallo Community,

wir wollen eine Spalte einer Kolibri-Tabelle, in der Datumswerten dargestellt werden, sortierbar machen. Mit dem Kolibri Datentyp für Tabellen-Zellen (KoliBriDataType = Record<string, unknown>;) geht das leider nicht, denn string wird lexikografisch sortiert, was bei Datumsangaben in deutschem Datumsformat als string zu falscher Sortierreihenfolge führt (bei englischem Datumsformat ginge es).
Was ist hier die best practice?

Datentyp für das _data soll KoliBriDataType oder string sein laut https://public-ui.github.io/docs/components/table/#properties
Viele Grüße, Andreas

**Comment by deleonio** [comment, marked as answer]
Hallo @AndyCooler,

hier ein Beispiel für Deine Anfrage: https://codesandbox.io/s/kolibri-public-ui-react-samples-w5u37c?from-embed=&file=/src/samples/table/sort-date.tsx&initialpath=/table/sort-date

Viele Grüße

**Reply by AndyCooler** [reply]
cool, danke
